protocol.md

```diff
@@ -1640,3 +1640,47 @@ interface RenameParams {
 _Response_
 * result: [`WorkspaceEdit`](#workspaceedit) describing the modification to the workspace.
 * error: code and message set in case an exception happens during the rename request.
+
+### Message overview
+
+* [initialize](#initialize) :leftwards_arrow_with_hook:
+* [shutdown](#shutdown) :leftwards_arrow_with_hook:
+* [exit](#exit) notification :arrow_right:
+* [$/cancelRequest](#cancelRequest) :arrow_right:
+
+Notifications server &rarr; client
+
+* [window/showMessage](#window_showMessage) notification :arrow_left:
+* [window/showMessageRequest](#window_showMessageRequest) :arrow_right_hook:
+* [window/logMessage](#window_logMessage) :arrow_left:
+* [telemetry/event](#telemetry_event) :arrow_left:
+* [textDocument/publishDiagnostics](#textDocument_publishDiagnostics) :arrow_left:
+
+Notifications client &rarr; server
+
+* [workspace/didChangeConfiguration](#workspace_didChangeConfiguration) :arrow_right:
+* [workspace/didChangeWatchedFiles](#workspace_didChangeWatchedFiles) :arrow_right:
+* [workspace/symbol](#workspace_symbol) :arrow_right:
```

Comment:
This is a request. It should be in the "Requests" section and the icon should be changed.